Introduction to Kotlin Multiplatform/06_jvm_render.md

```diff
@@ -1,33 +1,39 @@
-# Rendering in JavaScript
+# Common Rendering and JVM
```

Comment:
# Common Rendering and JVM

It is always good to question ourselves – do we really need the
JVM backend for the project? What if we could
run the [Mandelbrot Set](https://en.wikipedia.org/wiki/Mandelbrot_set)
rendering on the client? 

Let's implement the client-side rendering and see how what happens.